runtimes/standalone/src/lib.rs

```diff
@@ -771,15 +770,41 @@ pub type SignedExtra = (
 	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
 );
 /// Unchecked extrinsic type as expected by this runtime.
-pub type UncheckedExtrinsic = generic::UncheckedExtrinsic<Address, Call, Signature, SignedExtra>;
+pub type UncheckedExtrinsic = generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 /// The payload being signed in transactions.
-pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
+pub type SignedPayload = generic::SignedPayload<RuntimeCall, SignedExtra>;
 /// Extrinsic type that has already been checked.
-pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, Call, SignedExtra>;
+pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 /// Executive: handles dispatch to the various Pallets.
 pub type Executive =
 	frame_executive::Executive<Runtime, Block, frame_system::ChainContext<Runtime>, Runtime, AllPalletsWithSystem, ()>;
 
+#[cfg(feature = "runtime-benchmarks")]
+#[macro_use]
+extern crate frame_benchmarking;
```

Comment:
Interesting.. Why do we need this here?

Reply:
Can't this simply be done inside the `benche` module, since everything only exist for `runtime-benchmarks`?

Reply:
We're just following upstream here. See [this comment](https://github.com/paritytech/substrate/pull/10592#discussion_r787599647).

Reply:
Can we add a comment in code why we have this there?

Reply:
[f802641](https://github.com/KILTprotocol/kilt-node/pull/434/commits/f802641a4b2686693c25b18304ba31e47f9ed1da)